src/test/java/teammates/ui/webapi/GetFeedbackSessionsActionTest.java

```diff
@@ -190,7 +190,7 @@ protected void testExecute_asStudentWithCourseId_shouldReturnAllSessionsForCours
 
     }
 
-    @Test
+    @Test(enabled = false)
```

Comment:
Any particular reason we are ignoring these tests? Doesn't seem like a test that should fail.